conjur/wrapper/keystore_adapter.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+
+"""
+KeystoreAdapter module
+
+This module is a wrapper for the third-party keyring module used to get and set
+credentials in a system's keyring. This module allows us to easily swap out both
+library and functionality should we need to do so in the future.
+"""
+
+# Third party
+import keyring
+
+# Internals
+from conjur.constants import KEYSTORE_ATTRIBUTES
+
+
+class KeystoreAdapter:
```

Comment:
- should verify we are using the exact keyring version wanted.
- disable insecure keyring such as PlaintextKeyring.